code/style.css

```diff
@@ -1,4 +1,231 @@
+/* 
+colors 
+#F8EDE3
+#BDD2B6
+#A2B29F
+#798777 */
+
+*{ 
+   /* border: 1px solid black; */
+  margin: 0;
+}
+
+
 html {
-  background: #0025ff;
+  background: #F8EDE3;
   color: #fff;
 }
+
+nav{
+  display: none;
+}
+
+h1 {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
You have the same font family later on in the CSS, maybe it could be under the html tag :)